Microsoft.NET.Build.Containers/Tasks/CreateNewImage.Interface.cs

```diff
@@ -114,6 +114,15 @@ partial class CreateNewImage
     [Required]
     public string RuntimeIdentifierGraphPath { get; set; }
 
+    /// <summary>
+    /// The username or UID which is a platform-specific structure that allows specific control over which user the process run as.
+    /// This acts as a default value to use when the value is not specified when creating a container.
+    /// For Linux based systems, all of the following are valid: user, uid, user:group, uid:gid, uid:group, user:gid.
+    /// If group/gid is not specified, the default group and supplementary groups of the given user/uid in /etc/passwd and /etc/group from the container are applied.
+    /// If group/gid is specified, supplementary groups from the container are ignored.
+    /// </summary>
+    public string ContainerUser { get; set; }
```

Comment:
unless I'm missing something, this should be added to `CreateNewImageToolTask` to be passed to `containerize`.

Reply:
Excellent catch - I added it to the exe and didn't plumb it through here.

Reply:
Though I think we're missing a merge/rebase in the ToolsTask on this branch - I think you put a lot of changes in there recently?